eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -70,6 +70,13 @@
  * their behaviour and results.
  * NOT thread-safe.
  * @since 0.28.12
+ * @todo #2406:30min Fix {@link FakeMaven#allowedParams(Class)}
+ *  This function parameters of executed Mojo in {@link FakeMaven#execute(Class)}
```

Comment:
@Yanich96 I think English needs to be corrected here. Please use translator or chatGPT

Reply:
I have changed